package.json

```diff
@@ -40,6 +40,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
+    "docker": "docker-compose -f docker/docker-compose.yml up --build -d",
```

Comment:
This command could be confused and mistaken for a command to build Yorkie House into a docker image.